include/nuttx/streams.h

```diff
@@ -40,6 +40,16 @@
  * Pre-processor Definitions
  ****************************************************************************/
 
+#define stream_write(stream, buf, len) \
+        (( FAR struct lib_outstream_s *)stream)->puts \
+        (( FAR struct lib_outstream_s *)stream, buf, len)
+#define stream_putc(stream, buf, len) \
+        (( FAR struct lib_outstream_s *)stream)->putc \
+        (( FAR struct lib_outstream_s *)stream, ch)
+#define stream_getc(stream) \
```

Comment:
what other wrapper e.g. seek nput, nget

Reply:
What are the functions of nput and nget?

In all the code, I only see the value of modifying it, but no one has ever used it

Reply:
> What are the functions of nput and nget?
> 
> In all the code, I only see the value of modifying it, but no one has ever used it

it can be used in the future. Please add gets macro: https://github.com/apache/incubator-nuttx/pull/7681